docs/content/seo.md

```diff
@@ -0,0 +1,200 @@
+title = "Search Engine Optimization (SEO)"
+date = "2022-08-09T14:05:02.118466Z"
+
+[extra]
+url = "https://github.com/fermyon/bartholomew/blob/main/docs/content/seo.md"
+---
+
+## Google verification using Bartholomew
+
+Your Bartholomew implementation can be <a href="https://support.google.com/webmasters/answer/9008080?hl=en" target="_blank">officially verify with Google</a>. This is an important first step because the verification process provides you with access to the <a href="https://g.co/kgs/UcBemE" target="_blank">Google Search Console</a>. The Google Search Console (formlerly Google Webmaster Tools) allows webmasters to check indexing status, search queries, crawling errors and also optimize visibility of their site.
+
+Let's take a look at how the verification process is accomplished using Bartholomew. 
+
+### Markdown
+
+The first step in the Google verification process is where Google provides you (the owner) of the specific website with a specially named file i.e. `abcdefg.html`. Google now wants you, the owner, to make this file available on your site, so that Google can fetch it as proof that you are the site's owner which has access control to the site. This is a really simple task. First you create a Markdown file (in Bartholomew's `content` directory) called `abcdefg.md` (the name of this file just has to match the name of the file which Google provided). **Note: We are creating an `.md` file here not an `.html` file**.
+
+Bartholomew uses templating so you just have to be explicit about a couple of things inside that new `.md` file. Specifically,  Makes sure that there is a template name (we will create the template next) and that the content type (of this file) is rendered as `text/html`. This is shown in the source code of the new `abcdefg.md` file below.
+
+```
+title = "Google Verification"
+description = "Google verification file which provides us with access to Google Search Console"
+date = "2022-07-11T00:01:01Z"
+template = "google_verification"
+content_type = "text/html"
+---
+
+This is the abcdefg.html file that Google can see openly at the root of the website.
+```
+
+### Template
+
+Bartholomew uses <a href="https://handlebarsjs.com/" target="_blank">Handlebars</a> templating. Therefore the next step is for you to go ahead and create a new `google_verification.hbs` file in Bartholomew's template directory. Once the file is created, populate it with the content which Google requested, below is just an example.
+
+```
+{{! 
+For info on what can be placed here, see https://support.google.com/webmasters/answer/9008080#html_verification&zippy=%2Chtml-file-upload
+}}
+google-site-verification: abcdefg.html
+```
+
+At this point the verification button in the Google dashboard can be pressed; Google fetches the file from your site and the verification is complete!
```

Comment:
Does one leave the file here? Or after initial verification is it safe to delete?

Reply:
Ah, good question. My knowledge is that the file must remain available for Google to see/fetch on an ongoing basis.